env/prod/values.yaml

```diff
@@ -10,11 +11,16 @@ nginxingress:
   hosts:
     - name: release.k8s.p4.greenpeace.org
       tls: true
+      path: robots.txt
     - name: prod.p4.greenpeace.org
       tls: true
+      path: robots.txt
```

Comment:
Wouldn't also deploy [this robots.txt](https://github.com/greenpeace/planet4-robots/blob/main/public/robots.txt) on the root of `https://www.greenpeace.org/`?

Reply:
Not in this case. It only creates an ingress for host: prod.p4.greenpeace.org so if a user accesses https://prod.p4.greenpeace.org/robots.txt they get the right file otherwise if they access https://www.greenpeace.org/robots.txt they get a 404.